backend/src/controlers/login.handler.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express';
+import { User } from '../models/user.interface';
+import { compare } from 'bcrypt';
+import pkg from 'jsonwebtoken';
```

Comment:
Please use `import * as jwt from 'jsonwebtoken';`

Reply:
It's not working that way I've tried already last week